lib/holidays.rb

```diff
@@ -122,6 +122,40 @@ def between(start_date, end_date, *options)
       UseCaseFactory.between.call(start_date, end_date, date_driver_hash, regions, observed, informal)
     end
 
+    # Get next holidays occuring form date, inclusively.
+    #
+    # Returns an array of hashes or nil.
+    #
+    # Each holiday is returned as a hash with the following fields:
+    # [<tt>holidays_count</tt>]  Ruby Numeric object. This is the number of holidays to return
+    # [<tt>options</tt>]     One or more region symbols, <tt>:informal</tt> and/or <tt>:observed</tt>.
+    # [<tt>from_date</tt>]    Ruby Date object. This is an optional param, defaulted today.
+    #
+    # ==== Example
+    #   Date.today
+    #   => Tue, 23 Feb 2016
+    #
+    #   regions = [:us, :informal]
+    #
+    #   Holidays.next_holiday(3, regions)
+    #   => [{:name => "St. Patrick's Day",...},
+    #       {:name => "Good Friday",...},
+    #       {:name => "Easter Sunday",...}]
+    def next_holiday(holidays_count, options, from_date = Date.today)
```

Comment:
I really like the default of `Date.today`. I think that the idea of calling this method and not specifying a date and having it assume the current date is a great one.

I'm not 100% sold on this method name, though. How about we make it plural? So `next_holidays` instead. Minor change but I think we should make it clear that it can return multiple holidays (potentially).